.changeset/chilly-queens-call.md

```diff
@@ -0,0 +1,11 @@
+---
+"wrangler": minor
```

Comment:
I feel like in semver this is a minor release (adding a feature), but do we want to make it a patch release instead?

Reply:
Yeah, we're doing new features as patches